src/Renderer/ProjectiveTexturingMaterial.js

```diff
@@ -241,14 +241,16 @@ var ProjectiveTexturingMaterial = {
 
         _infos.pano = panoInfo;
         _infos.lod = _infos.lods[0];
-        for (i = 0; i < N; ++i) {
+        for (let i = 0; i < N; ++i) {
             _infos.cam = Ori.sensors[i].infos; // console.log(_infos.cam);
-            m = idmask[i];
+            const m = idmask[i];
             if (m >= 0) {
+                // eslint-disable-next-line no-loop-func
                 this.loadTexture(Ori.getMask(i), {}, (tex, m) => {
                     _shaderMat.uniforms.mask.value[m] = tex;
                 }, m);
             }
+            // eslint-disable-next-line no-loop-func
```

Comment:
I did

```diff
+        function setMaskOnLoad(tex, m) {
+            _shaderMat.uniforms.mask.value[m] = tex;
+        }
+
+        function setTextureOnLoad(tex, i) {
+            _shaderMat.uniforms.texture.value[i] = tex;
+        }
+
         _infos.pano = panoInfo;
         _infos.lod = _infos.lods[0];
         for (i = 0; i < N; ++i) {
             _infos.cam = Ori.sensors[i].infos; // console.log(_infos.cam);
             m = idmask[i];
             if (m >= 0) {
-                this.loadTexture(Ori.getMask(i), {}, (tex, m) => {
-                    _shaderMat.uniforms.mask.value[m] = tex;
-                }, m);
+                this.loadTexture(Ori.getMask(i), {}, setMaskOnLoad, m);
             }
-            this.loadTexture(_infos.url, _infos, (tex, i) => {
-                _shaderMat.uniforms.texture.value[i] = tex;
-            }, i);
+            this.loadTexture(_infos.url, _infos, setTextureOnLoad, i);
         }
         this.changePanoTextureAfterloading(panoInfo, posFrameWithPivot, rot, 1);
```

If I'm not mistaken, the only variables captured by these closures are from outside the loop. So putting the function declaration outside of it should cause no problems.